frontend/src/components/form/BirthField/BirthField.tsx

```diff
@@ -1,14 +1,26 @@
+import DatePicker, { ReactDatePickerProps, registerLocale } from "react-datepicker";
 import classNames from "classnames";
 import ko from "date-fns/locale/ko";
-import PropTypes from "prop-types";
-import DatePicker, { registerLocale } from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
+
 import Label from "../../@common/Label/Label";
+
 import styles from "./BirthField.module.css";
 
 registerLocale("ko", ko);
 
-const BirthField = ({ value, onChange, required, className, ...props }) => {
+type BirthFieldProps = Omit<ReactDatePickerProps, "value"> & {
+  value?: Date | null;
```

Comment:
r: 디폴트 파라미터로 null을 할당하기 때문에 옵셔널 연산자는 제거하는게 어떨까요?

```suggestion
  value: Date | null;
```

Reply:
https://github.com/woowacourse/service-apply/pull/654#discussion_r1028726197